src/sentry/api/validators/sentry_apps/schema.py

```diff
@@ -88,6 +87,31 @@
                 'name': {
                     'type': 'string',
                 },
+                'default': {
+                    'type': 'string',
+                    'enum': ['issue.title', 'issue.description'],
+                },
+            },
+            'required': ['type', 'label', 'name'],
+        },
+
+        'textarea': {
+            'type': 'object',
+            'properties': {
+                'type': {
+                    'type': 'string',
+                    'enum': ['textarea'],
+                },
+                'label': {
+                    'type': 'string',
+                },
+                'name': {
+                    'type': 'string',
+                },
+                'default': {
+                    'type': 'string',
+                    'enum': ['issue.title', 'issue.description'],
```

Comment:
If we end up using this same sort of thing in another form field, we can pull it out into it's own `definition`. It's fine for now since it's only in two places, though.